apps/user-office-frontend/src/components/SEP/Proposals/SEPProposalsAndAssignmentsTable.tsx

```diff
@@ -114,7 +127,7 @@ const SEPProposalColumns = [
 
       return isNaN(stdDeviation) ? '-' : `${stdDeviation}`;
     },
-    customSort: (a: SEPProposalType, b: SEPProposalType) =>
+    customSort: (a, b) =>
```

Comment:
What was the reason for removing the types? Are these types inferred?

Reply:
Yea, I did some cleanup for types here. I assigned the type at the top level, so it won't require types for argument of the render method. You can still check the types when you hover the arg.